resmio-api/resmio-api.php

```diff
@@ -269,36 +269,44 @@ function resmio_api_data_update() {
 	if( isset( $options['api_descr_short'] ) ) { $apiDescrShrt = $options['api_descr_short']; } else { $apiDescrShrt = ''; }
 
 	if( $get_api_data == FALSE ):
-	echo '<div class="error fade"><p><strong>'.__('Ungültige ID', 'resmio_i18n').'</strong></p></div>'; 
+	echo '<div class="error fade"><p><strong>'.__('Ungültige ID', 'resmio_i18n').'</strong></p></div>';
 	echo '<p><strong>'.__('Noch keine resmio ID? Melden Sie sich <a href="https://www.resmio.com" target="_blank">hier</a> kostenlos an!', 'resmio_i18n').'</strong></p><br>';
-
-	else: 
 	?>
-	<body onload="additionalOpenHours()">
```

Comment:
A body tag inside a body tag is only allowed if the second body tag is nested inside an iframe and instead of '<body onload="additionalOpenHours()">' you can use than the more convenient '$( document ).ready(function () {});' (line 291)